### ISSUE TITLE
Form Validation -> Assignment:  Add another section to skip in the 2nd assignment

### ISSUE THREAD

**Issue by shonebinu** [TheOdinProject/curriculum#25082]
### Describe your suggestion

In the 2nd assignment of the assignment section, the guide tells us to skip over the section "JavaScript and the Constraint validation API" of the article "https://www.sitepoint.com/html-forms-constraint-validation-complete-guide/", since it hasn't covered thoroughly before. But, the section that's coming after the given section in the same article, which is "Creating a Custom Form Validator" is hard to follow and the example codes are super confusing since it's not been covered before. I think the Odin authors didn't intend for the readers to study that part. 

I'm in a trance since I'm not sure if it's material that I have to follow or not. If it's not meant to be, I suggest adding a line to skip that section as well.
Please pardon me if I'm making any mistakes in this Issue. New to this game.

### Path

Node / JS

### Lesson Url

https://www.theodinproject.com/lessons/node-path-intermediate-html-and-css-form-validation#assignment

### Checks

- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

**Comment by wise-king-sullyman** [comment]
Indeed it looks like maybe the article was updated or something since we linked to it?

@KevinMulhern it looks like you wrote this lesson, was the intent for that section to be read for some other reason?

**Comment by codyloyd** [comment]
I concur with @wise-king-sullyman and the OP.

should tweak the lesson to also skip that paragraph.  I'll do it.
